webpack.config.test.js

```diff
@@ -1,21 +1,26 @@
 const nodeExternals = require('webpack-node-externals');
 const path = require('path');
 
-module.exports = {
-    mode: 'development',
-    entry: './src/index.ts',
-    resolve: {
-        extensions: ['.ts'],
-    },
-    module: {
-        rules: [{
-            test: /\.ts$/,
-            exclude: /node_modules/,
-            include: path.resolve('src'), // instrument only testing sources with Istanbul, after ts-loader runs
-            use: ['istanbul-instrumenter-loader', 'ts-loader']
-        }],
-    },
-    target: 'node',  // webpack should compile node compatible code
-    externals: [nodeExternals()], // in order to ignore all modules in node_modules folder
-    devtool: 'inline-cheap-module-source-map'
+module.exports = (env, argv) => {
+    const isDebug = env == 'dbg';
+    const ifDebug = (whenDebug, whenNot) => (isDebug ? whenDebug : whenNot);
+
+    return {
+        mode: 'development',
+        entry: './src/index.ts',
+        resolve: {
+            extensions: ['.ts'],
+        },
+        module: {
+            rules: [{
+                test: /\.ts$/,
+                exclude: /node_modules/,
+                include: path.resolve('src'), // instrument only testing sources with Istanbul, after ts-loader runs
+                use: ifDebug('ts-loader', ['istanbul-instrumenter-loader', 'ts-loader'])
```

Comment:
I recommend to use Webpack 4 modes instead.

Reply:
As far as I can see, mode only supports `development` and `production`: https://webpack.js.org/concepts/mode/

But for this case I wanted to have different ways for `development`. The reason is the `istanbul-instrumenter-loader`. If it is active, breakpoints don't work any more due to issues with source maps.

Reply:
Ok, let's continue with your implementation then. I can't provide any better options at the moment.